ouroboros-consensus-byronspec/src/Ouroboros/Consensus/ByronSpec/Ledger/Block.hs

```diff
@@ -5,12 +5,13 @@
 {-# LANGUAGE TypeFamilies      #-}
 
 module Ouroboros.Consensus.ByronSpec.Ledger.Block (
```

Comment:
The way it moved the comment is a little confusing. Is there a clever way to resolve that?

This example of "type alias" is not crucial, but IIRC I'll see some below that are little more useful (eg I'm anticipating "opaque").

Worst-case: we can promote the important such comments from comments to relevant Haddock sections.

Reply:
Yeah, this is definitely a bug. I've [created an issue in stylish-haskell](https://github.com/jaspervdj/stylish-haskell/issues/359). 
For now, do you want time to go through all occurrences of this issue and replace comments with haddock comments? I'm not really looking forward to this :) but then again.. I gave you a 350+ PR to review... :)

Reply:
We rephrased these annotations via Haddock sections.